### PR TITLE
Enable some tests in Java

### PR DESCRIPTION
**Pull request by smola**
## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com
* [ ] if a scenario is added (or removed), add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
Ci is failing : https://github.com/DataDog/system-tests/actions/runs/6014044212/job/16313026652?pr=1538#step:14:200

File: tests/appsec/test_reports.py [hunk 1]
```diff
@@ -24,7 +24,9 @@
 @released(golang={"gin": "1.37.0", "echo": "1.36.0", "*": "1.34.0"})
 @bug(library="python@1.1.0", reason="a PR was not included in the release")
 @missing_feature(weblog_variant="akka-http", reason="No AppSec support")
-@missing_feature(weblog_variant="spring-boot-payara", reason="No AppSec support")
+@missing_feature(
```
Comment (cbeauchesne) [inline tests/appsec/test_reports.py:27]: Can you remove the java declaration on line 23 and use this : 

```suggestion
@released(java={"*":"0.92.0", "spring-boot-payara":"1.6.0"})
```

(or better, use manifest file)

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/appsec/test_reports.py (hunk 1 above)
Comment (smola) [reply tests/appsec/test_reports.py]: Done. If you don't mind, I'd prefer to migrate existing tests to the manifest in separate PRs, as these moves are relevant to more languages.

**Review by cbeauchesne — COMMENTED** [review]
CI is failing, but I think it's more instability of the weblog app at startup : 

https://github.com/DataDog/system-tests/actions/runs/6081999251/job/16533936071?pr=1538#step:32:27

Let me check the log

**Comment by cbeauchesne** [comment]
Not related, waiting for #1566 to fix it.

**Review by cbeauchesne — APPROVED** [review]

**Comment by cbeauchesne** [comment]
Hopefully, it'll the last massive conflict to handle... Sorry for the trouble !

File: manifests/java.yml [hunk 2]
```diff
@@ -352,8 +358,10 @@ tests/:
         spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
         spring-boot-jetty: v0.111.0
         spring-boot-openliberty: v0.115.0
-        spring-boot-payara: missing_feature (Missing support)
+        spring-boot-payara: v1.7.0  # Exact version for Payara unclear
```
Comment (cbeauchesne) [inline manifests/java.yml:361]: CI is failing on this one

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

**Review by cbeauchesne — APPROVED** [review]
<del>Just a request change, on unused `_released_java_blocking`.
I approve it, you can merge it once it's removed. Don't mind about python failure, it's not related.</del>

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
Actually, undoing my approval, as sone scenario were missing in CI

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**